shared/js/views.jsx

```diff
@@ -547,7 +547,7 @@ loop.shared.views = (function(_, mozL10n) {
       var hostname;
 
       try {
-        hostname = new URL(this.props.url).hostname;
+        hostname = loop.shared.utils.formatURL(this.props.url, true).hostname;
```

Comment:
had to leave in as it returns null for invalid url we could have used an if statement also.